README.md

```diff
@@ -74,6 +74,10 @@ $ rails db:create
 When you will add migration file then, you can add command like
 ```
 $ rails db:migrate
+```
+Run this command line to add some dummy data
+```
+$ rails db:seed
```

Comment:
- [ ] I would recommend adding the command `npm i` to install the npm packages because without this the application is not running and giving errors. Please add the command.

Reply:
I already noticed that in the PR description 👍🏼